patroni/postmaster.py

```diff
@@ -138,7 +138,8 @@ def start(pgcommand, data_dir, conf, options):
         # of init process to take care about postmaster.
         # In order to make everything portable we can't use fork&exec approach here, so  we will call
         # ourselves and pass list of arguments which must be used to start postgres.
-        env = {p: os.environ[p] for p in ('PATH', 'LD_LIBRARY_PATH', 'LC_ALL', 'LANG') if p in os.environ}
+        # On Windows, in order to run a side-by-side assembly the specified env must include a valid SYSTEMROOT.
+        env = {p: os.environ[p] for p in ('PATH', 'LD_LIBRARY_PATH', 'LC_ALL', 'LANG', 'SYSTEMROOT') if p in os.environ}
```

Comment:
Shouldn't `SYSTEMROOT` also be added to https://github.com/zalando/patroni/blob/master/patroni/postgresql.py#L1225 ?

Please also make sure that you provide the correct env that `pg_controldata` speaks English.

Reply:
`LC_ALL` seems enough for speaking English